src/AST/Validation/Tag.cc

```diff
@@ -67,4 +70,20 @@ bool Tag::_validateSingleNumeric(const ASTNode *node) {
     return true;
 }
 
+bool Tag::_validateList(const ASTNode *node) {
+    const auto *tag = dynamic_cast<const ASTTag*>(node);
+    const ASTNode *parent = tag->parent();
```

Comment:
nit: probably don't need parent as it's not used

Reply:
Would the preference be to collapse it into `next`? 
e.g.
```c++
    const ASTNode *next = tag->parent()->nextChild(tag);
```

It follows the pattern of the other private members in `Tag`

Reply:
Oh nevermind, I didn't see that. It's fine as-is then.

Reply:
No worries! It could be condensed, but I wonder if `parent` should have a `nullptr` check. Semantically, as far as I understand it, being a `nullptr` _should_ never happen..